src/components/victory-bar.jsx

```diff
@@ -101,6 +109,16 @@ export default class VictoryBar extends React.Component {
      */
     categoryLabels: React.PropTypes.array,
     /**
+     * The colorScale prop is an optional prop that defines the color scale the chart's bars
+     * will be created on. This prop should be given as a string, which will one of the five
+     * baked-in color scales: "victory", "grayscale", "red", "bluePurple", and "yellowBlue".
+     * If it is not defined, the default Victory grayscale will be used, and if the fill
+     * property on the dataAttributes prop is defined, it will overwrite the colorScale prop.
+     */
+    colorScale: React.PropTypes.oneOf([
```

Comment:
yeah--as a user, I'd probably prefer this. Right now, dataAttributes.fill takes precedence over colorScale, and I'm leaning slightly toward keeping that even as the user can set their own colorScale array. do you have strong preferences either way?